roombapy/discovery.py

```diff
@@ -17,21 +17,15 @@ class RoombaDiscovery:
     udp_port = 5678
     roomba_message = "irobotmcs"
     amount_of_broadcasted_messages = 5
-    server_socket = None
-    log = None
+    server_socket: socket.socket
+    log: logging.Logger
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize the discovery class."""
         self.server_socket = _get_socket()
         self.log = logging.getLogger(__name__)
```

Comment:
This can be set outside of the constructor

Reply:
But it should not, as this methods are not pure (i.e. they are stateful).